R/prep_sankey.R

```diff
@@ -92,40 +92,39 @@ check_prep_sankey <- function(data_alignment,
                               middle_node2) {
   names_all <- c(group_var, "name_abcd", "sector")
   names_aggergate <- c("region", "year")
-  abort_if_missing_names(data_alignment, c(names_all, names_aggergate))
+  assert_missing_names(data_alignment, c(names_all, names_aggergate))
   if (!(region %in% unique(data_alignment$region))) {
-    rlang::abort(c(
-      "`region_tms` value not found in `data_alignment` dataset.",
-      i = glue::glue("Regions in `data_alignment` are: {toString(unique(data_alignment$region))}"),
-      x = glue::glue("You provided region = {region}.")
+    cli::cli_abort(c(
+      x = "{.arg region} value not found in {.var data_alignment} dataset",
+      i = "{cli::qty(length(unique(data_alignment$region)))}region{?s} in
+           {.var data_alignment} {?is/are}:
+          {.val {as.character(unique(data_alignment$region))}}",
+      i = "the value{?s} in {.arg region} {?is/are}: {.val {region}}"
     ))
   }
   if (!(year %in% unique(data_alignment$year))) {
-    rlang::abort(c(
-      "`year` value not found in `data_alignment`.",
-      i = glue::glue(
-        "Years in `data_alignment` are: {toString(unique(data_alignment$year))}
-        "
-      ),
-      x = glue::glue("You provided year = {year}.")
+    cli::cli_abort(c(
+      x = "{.arg year} value not found in {.var data_alignment} dataset",
+      i = "{cli::qty(length(unique(data_alignment$year)))}year{?s} in
+           {.var data_alignment} {?is/are}: {.val {unique(data_alignment$year)}}",
+      i = "the value{?s} in {.arg year} {?is/are}: {.val {unique(year)}}"
     ))
   }
-  abort_if_middle_node_column_not_found(data_alignment, middle_node, env = list(data = substitute(data_alignment)))
+  assert_middle_node_column_not_found(data_alignment, middle_node, env = list(data = substitute(data_alignment)))
```

Comment:
shouldn't the naming be something like `assert_middle_node_column_exists()`?

Reply:
done